client/app.jsx

```diff
@@ -63,41 +65,111 @@ const AuthorName = styled.span`
+  font-weight:250;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  `;
+
+const SmallIcon = styled.svg`
+  width: 14px;
+  box-sizing: border-box;
+  display: inline;
+  `;
+
+const InfoText = styled.span`
+  margin-left: 8px;
+  display: inline;
+  `;
+
+const Wishlist = styled.button`
+  position: relative;
+  color: white;
+  height: 40px;
+  border: 1px solid white;
+  font-weight: 700;
+  background: black;
+  align-items: center;
+  border-radius: 4px;
+  display: inline-flex;
+  min-width: 10rem;
+  padding: 0 1.2rem;
+  `;
+
+const HeartIcon = styled.svg`
+  width: 24px;
+  height: 24px;
+  `;
+
 class Overview extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       overview: {},
       review: {
-        average: 3.7,
-        total: 16384
-      } // hard-coded until API is running
+        average: 0,
+        total: 0
+      },
+      updatedAt: '1-1-2021',
+      wishlist: false
     };
-  };
-
-  capitalize(string) {
-    return string.slice(0, 1).toUpperCase() + string.slice(1);
+    this.wishClick = this.wishClick.bind(this);
   }
+
   componentDidMount() {
     const regex = /\d+/;
     let course = window.location.search.match(regex) === null ? 5 : window.location.search.match(regex)[0];
     console.log(course);
+    console.log('Updated at', moment(this.state.updatedAt).format('M/YYYY'));
     this.getOverview(course);
   }
 
   getOverview(id = 5) {
     axios.get(`http://localhost:3000/overview/?courseId=${id}`)
-    .then((res) => {
-      let overview = res.data;
-      let review = this.state.review;
-      this.setState({
-        overview: overview
-      });
-      //will need further API calls here /-- these should be calls to external components
-    })
-    .catch((err) => console.log(err));
+      .then((res) => {
+        let overview = res.data;
+        let review = this.state.review;
+        this.setState({
+          overview: overview
+        });
+        //will need further API calls here /-- these should be calls to external components
+      })
+      .then(() => {
+        axios.get(`http://localhost:2712/reviews/item?courseId=${id}`)
+        .then((res) => {
+          console.log(res.data);
+          let overallRating = res.data.ratings.overallRating;
+          let totalRatings = res.data.ratings.totalRatings;
+          this.setState({
+            review: {
+              average: overallRating,
+              total: totalRatings
+            }
+          });
+        })
+        .then(() => {
+          axios.get(`http://ec2-18-130-234-175.eu-west-2.compute.amazonaws.com:9800/course/item?courseId=${id}`)
+          .then((res) => {
+            let updatedAt = res.data.updatedAt;
+            this.setState({
+              updatedAt: updatedAt
+            });
+          })
+          .catch((err) => console.log(err));
+        })
+        .catch((err) => console.log(err));
+      })
+      .catch((err) => console.log(err));
+  }
+
```

Comment:
if there is a bug, it might be easier to find it if each step had its own function?

Reply:
Good call! I could at least prepend something to the console.log -- something like console.log('Error in API call: ' + err);